library/spdm_common_lib/libspdm_com_context_data.c

```diff
@@ -2071,9 +2071,10 @@ libspdm_return_t libspdm_init_context_with_secured_context(void *context,
     libspdm_context_t *spdm_context;
     size_t index;
 
-    LIBSPDM_ASSERT(context != NULL);
-    LIBSPDM_ASSERT(secured_contexts != NULL);
-    LIBSPDM_ASSERT(num_secured_contexts == LIBSPDM_MAX_SESSION_COUNT);
+    if (context == NULL || secured_contexts == NULL ||
```

Comment:
The caller should guarantee those conditions are satisfied before call this function.

Can we fix the caller?